### PR TITLE
Simplify section "Boot a live desktop image to fix..." in fix-broken-install

### LINKED ISSUE #948: Revise Sphinx include and simplify text in fix-broken-install
**Describe the error/improvement to an existing document or image**
See: https://docs.01.org/clearlinux/latest/guides/maintenance/fix-broken-install.html?highlight=repair

### PR DESCRIPTION
**Pull request by mvincerx**
- Closes #948  
- Remove include per request from @bktan8
- Make concise the Prereq's

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/guides/maintenance/fix-broken-install.rst [hunk 1]
```diff
@@ -17,25 +16,23 @@ This guide assumes you have installed |CL| on a target system, but the OS
 does not boot or function properly.
 
 The process described in this guide can only verify and fix files that
-:ref:`swupd<swupd-guide>` owns in :file:`/usr`. Files outside of this path, such
-as :file:`/home/`, :file:`/etc`, :file:`/var`, etc., cannot be repaired by this
-process.
+:ref:`swupd<swupd-guide>` owns in :file:`/usr`. Files outside of this path,
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]:
```suggestion
:ref:`swupd<swupd-guide>` owns in :file:`/usr` and :file:`/var`. Files outside of this path, 
```

File: source/guides/maintenance/fix-broken-install.rst [hunk 2]
```diff
@@ -17,25 +16,23 @@ This guide assumes you have installed |CL| on a target system, but the OS
 does not boot or function properly.
 
 The process described in this guide can only verify and fix files that
-:ref:`swupd<swupd-guide>` owns in :file:`/usr`. Files outside of this path, such
-as :file:`/home/`, :file:`/etc`, :file:`/var`, etc., cannot be repaired by this
-process.
+:ref:`swupd<swupd-guide>` owns in :file:`/usr`. Files outside of this path, 
+such as :file:`/home/`, :file:`/etc`, :file:`/var`, etc., cannot be repaired
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]:
```suggestion
such as :file:`/home/`, :file:`/etc`, etc., cannot be repaired
```

File: source/guides/maintenance/fix-broken-install.rst [hunk 3]
```diff
@@ -17,25 +16,23 @@ This guide assumes you have installed |CL| on a target system, but the OS
...
+by this process.
 
 Prerequisites
 *************
 
-* Download and install the live desktop image on a USB. See
-  :ref:`bare-metal-install-desktop` for install instructions.
+* Download and install the live desktop image on a USB.
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]:
```suggestion
* Download and burn the live desktop image on a USB. 
```

File: source/guides/maintenance/fix-broken-install.rst (hunk 2 above)
Comment (mvincerx) [reply source/guides/maintenance/fix-broken-install.rst]: Done

File: source/guides/maintenance/fix-broken-install.rst (hunk 3 above)
Comment (mvincerx) [reply source/guides/maintenance/fix-broken-install.rst]: Done

File: source/guides/maintenance/fix-broken-install.rst (hunk 1 above)
Comment (mvincerx) [reply source/guides/maintenance/fix-broken-install.rst]: Done

**Comment by mvincerx** [comment]
@bktan8, see latest commit after CI completes.

**Pull request merged**